Smooth UI to --log2

Jessime opened the pull request:
From an end user's perspective, the only thing I've really done here is make the parameter options for `--log2` be one of `['pre', 'post', 'none']`

I also fooled around with the documentation a little, but nothing substantial. @spragud2, I didn't delete anything you wrote, just moved it around.

Commit:
Smooth UI to --log2

jmcalabr commented:
Jessime that makes sense to me – can one of you edit the README to be consistent with the new option?

Thank you both for your help –
Mauro

From: Jessime Kirk <notifications@github.com>
Reply-To: CalabreseLab/seekr <reply@reply.github.com>
Date: Tuesday, July 21, 2020 at 2:24 AM
To: CalabreseLab/seekr <seekr@noreply.github.com>
Cc: "Calabrese, Mauro" <mauro_calabrese@med.unc.edu>, Review requested <review_requested@noreply.github.com>
Subject: [CalabreseLab/seekr] Smooth UI to --log2 (#15)


From an end user's perspective, the only thing I've really done here is make the parameter options for --log2 be one of ['pre', 'post', 'none']

I also fooled around with the documentation a little, but nothing substantial. @spragud2<https://github.com/spragud2>, I didn't delete anything you wrote, just moved it around.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/CalabreseLab/seekr/pull/15

Commit Summary

  *   Smooth UI to --log2

File Changes

  *   M CHANGELOG.md<https://github.com/CalabreseLab/seekr/pull/15/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e> (25)
  *   A CONTRIBUTING.md<https://github.com/CalabreseLab/seekr/pull/15/files#diff-6a3371457528722a734f3c51d9238c13> (32)
  *   M README.md<https://github.com/CalabreseLab/seekr/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8> (19)
  *   M seekr/__version__.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-43ddbfcac041c8355be720f865c72677> (2)
  *   M seekr/console_scripts.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-c1428071047eb87e2136b1416c84627e> (32)
  *   M seekr/kmer_counts.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-46561d1dc3c61dbaa9a77dc765584946> (22)
  *   M seekr/pearson.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-ac3d36f3aaad1dbb2a6756ee118dfaa6> (9)
  *   M seekr/tests/test_console_scripts.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-d46ba02877cb20da57953776088526cd> (9)
  *   M seekr/tests/test_kmer_counts.py<https://github.com/CalabreseLab/seekr/pull/15/files#diff-87dedc1022069e31ba30dbc68ade70d8> (2)

Patch Links:

  *   https://github.com/CalabreseLab/seekr/pull/15.patch
  *   https://github.com/CalabreseLab/seekr/pull/15.diff

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://github.com/CalabreseLab/seekr/pull/15>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALDCYUU4HTGEU36SQ36DBZTR4UYBZANCNFSM4PDGQC3A>.

Commit:
Update --log2 usage in README

Also move some of Dan's longer notes to the bottom.

Jessime commented:
@jmcalabr Good catch, thank you. I updated the README in [this commit](https://github.com/CalabreseLab/seekr/pull/15/commits/4215490dab9ba7a5712cf5bf6948ece61f34aa03) and also moved a couple of Dan's longer comments down towards the bottom. (Like before, didn't delete anything, just some minor shuffling).

Since the interface has been approved, I'm going to go ahead and merge this PR, and make a new release available on pypi.  We can continue making changes either in the master branch (for small doc related issues) or in a new branch (for larger things) if needed.
